### DISCUSSION TITLE
Unable to pull in component ". Radio (single row)"

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#1653] [Q&A] [answered]
Hi! I was attempting to pull in the ". Radio (single row)" component to a design, but am unable to. I understand the "." usually means WIP/private. Happy to use the "Radio Group" component and delete out the parts I don't need, just wanted to double check that the single version was unavailable. Thanks!

**Comment by serifluous** [comment, marked as answer]
@ceyerkes Thanks for the question! Yep, you're right in that it's meant to be private to the library file. Single rows of radios shouldn't really be used since they're meant to sit in a group and [let users select one thing among multiple options](https://paste.twilio.design/components/radio-group#about-radio-group).

I'd love to hear about your use case, though! Can you share a screenshot/Figma file? We might be able to point you toward another component or solution.

**Reply by ceyerkes** [reply]
@serifluous gotcha, that makes sense - thank you! 

I'm in the process of migrating/rebuilding some Sketch files in Figma, and ran into this console modal with a single line of radio buttons:

![Screen Shot 2021-07-08 at 11 45 12 AM](https://user-images.githubusercontent.com/87141551/124967711-f66e6e00-dfe1-11eb-8022-eaf373d74091.png)

Please let me know if there's another component/solution that you would recommend, or if I should just use the radio group component, thanks!

**Reply by kate-porter** [reply]
Hey @ceyerkes - you've just found a gap that we overlooked when creating the Figma components! In Paste, it is totally possible to have Radio Groups and Checkboxes sit horizontally like you have in your screenshot, but we overlooked adding that horizontal layout as a variant in Figma. We've added a ticket to our backlog to add that, but in the meantime, using the `Radio Group` and deleting the parts you don't need is the way to go. Thanks for catching this!

**Reply by ceyerkes** [reply]
@katieporter sounds good, thank you!
